docs/examples/nodejs-read-versioned-layer.md

```diff
@@ -76,39 +77,41 @@ Log into the api. Modify our app to the following state:
 /**
  * Example Node.js app for reading versioned layer from datastore
  */
-import { UserAuth } from "@here/olp-sdk-authentification";
+import { UserAuth, requestToken } from "@here/olp-sdk-authentification";
 
 const userAuth = new UserAuth({
     env: "here",
     credentials: {
         accessKeyId: "your-access-key-id",
         accessKeySecret: "your-access-key-secret"
-    }
+    },
+    tokenRequester: requestToken
 });
 
 ```
 
-## Context
+## OlpClientSettings
 
 Now you can initialize DataStore client for `here` environment and, for example, list catalogs.
-To do so you need a `DatastoreContext`. It contains look up api functionallity to get the base urls to the services.
+To do so you need a `OlpClientSettings`. It contains look up api functionallity to get the base urls to the services.
```

Comment:
OlpClientSettings does not contain lookup api func... . It contains download manager, token provider, and key-value cache

Reply:
Fixed